schemas/JSON/settings/settings.schema.0.2.json

```diff
@@ -91,12 +91,24 @@
         }
       }
     },
+    "InstallRoot": {
+      "description": "The root directory where packages are installed to. Applies to the portable installer type.",
+      "type": "string",
+      "default":  "%LOCALAPPDATA%/WinGet"
+    },
+    "AddToPathEnvironment": {
+      "description": "Boolean value indicating whether the installed location should be added to the PATH environment. Applies to the portable installer type.",
```

Comment:
I thought we were adding to the App Path registry?  I think we should hold off on supporting %PATH% updates to see if App Path is good enough.